src/session/content/message_row/indicators_model.rs

```diff
@@ -111,4 +154,34 @@ impl MessageIndicatorsModel {
 
         String::new()
     }
+
+    pub(crate) fn sending_state_icon_name(&self) -> String {
+        self.imp()
+            .message
+            .borrow()
+            .as_ref()
+            .and_then(|message| message.downcast_ref::<Message>())
+            .filter(|message| message.is_outgoing())
+            .map(|message| {
+                if message.chat().is_own_chat() {
+                    "message-read-symbolic"
```

Comment:
Here we should at least check the `MessageSendingState`, otherwise we always get the `read` icon even when we send this message with no internet connection.